RedHat-UX/red-hat-design-system
# docs(avatar): update docs

## What I did

1. Updated the docs for rh-avatar element based on the specs mentioned in https://github.com/RedHat-UX/red-hat-design-system/issues/909 


## Testing Instructions

1. Review the updated docs for [rh-avatar(preview)](https://deploy-preview-1002--red-hat-design-system.netlify.app/elements/avatar/) and check if they match the [design spec](https://xd.adobe.com/view/6f3078cf-c5d5-4c7b-8e30-509acd11525c-3a63/) and the [content](https://docs.google.com/document/d/1eEnyWKABadmP4RetzXIB7HXX542C9n-Dj_jjQ4uE_XI/edit?usp=sharing)
